source/blog/2016-09-05-last-week-in-manageiq-api-refactor-observables.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Last Week in ManageIQ: API refactor, Observables and more!"
+date: 2016-09-05
+author: himdel
+comments: true
+published: true
+---
+
+Greetings from sunny Brno, [happy Martin](https://github.com/himdel) here, from the UI team :).
```

Comment:
Fixed Happy.. I kinda prefer smileys and seems like the site ignores :cookie: and stuff.. so not quite sure about emojis :)